### PR TITLE
stripped down tutorial intro no CSS

### PR DESCRIPTION
**Pull request by davidjohnbaker1**
First attempt at seeing if I can help out more on the small Congo related tasks.

Attempted #609 , with the following:

- Copied matching pairs to make matching pairs tutorial file
- Added in call to rules in init 
- removed unnecessary code other that start and do it again (looking at this level of minimalism for congo work)

Not sure if

- data will save, no need for it to, but have not totally figured out save structure

New Issues

- Will need to make a new issue to fix the CSS so it's a 2x3 grid, but didn't want to make a hack solution to this on the front end that just creates code duplication, will file this right away for future. If you have suggestions on best ways to do it (was going to just make Matching Pairs.js/scss duplicate tutorial set of files, then change all the calls from the new matching_pairs_tutorial.py file, but that didn't seem like the most elegant want to do it. 

Please let me know if these kinds of small fixes with my limited webdev skills are ultimately going to help out or just make more work for everyone. 

Also did this by cloning off main (as I got errors when I ran off of development).

### CODE REVIEW

File: backend/experiment/rules/matching_pairs_tutorial.py
Comment (BeritJanssen) [inline backend/experiment/rules/matching_pairs_tutorial.py:1]: This whole file can be reduced to three lines:
```
from .matching_pairs import MatchingPairs

class MatchingPairsTutorial(MatchingPairs):
    ID = 'MATCHING_PAIRS_TUTORAL'
    num_pairs = 3
```

No need to repeat any of the other code then, and it's possible to change the questions from the admin interface.

**Review by BeritJanssen — CHANGES_REQUESTED** [review]
Using inheritance, we can make the rules file for the MatchingPairsTutorial *much* shorter. Every method from the parent class will be reused, and you only only override the `ID` and `num_pairs class` variables.

**Comment by BeritJanssen** [comment]
Thanks for your reasoning to make a new issue. It would indeed not be desirable to make a new frontend component specifically to deal with less cards in a 2x3 grid.

**Comment by davidjohnbaker1** [comment]
Ah, more more elegant! 

Though for the tutorial, we do not want the intro text or all the stuff at the end, just a basic loop that goes :

ready to play -- six card game -- play again? 

This will never be used online, just by lia in the congo. 

I've amended the commit to reflect that, but defer to you on if what i've pushed is the best way to deal with this. 

Also, I was not able to get this code to run only with a call to 

`from .matching_pairs import MatchingPairs`

but did try to leave only the required library calls.

**Comment by drikusroor** [comment]
@davidjohnbaker1 @BeritJanssen I see this PR has been in a "Changes requested" state for over a month now. Is there anything currently blocking it from going forward?

**Comment by BeritJanssen** [comment]
As far as I'm aware, the "tutorial with fewer cards" feature was abandoned for now, so I believe we can close this?

**Comment by davidjohnbaker1** [comment]
Yes, this was abandoned. We mis-communicated about this, I will close the issue for now.

**Pull request closed without merging**